### PR TITLE
Pokedex attack improvement

### PR DESCRIPTION
**Pull request by mfkaptan**
- Wrap attack component into ion-card
- Add badges for types and numbers

### CODE REVIEW

**Review by AlexanderLill — APPROVED** [review]
Thanks :)

File: ionic2/app/components/poke-details/poke-attacks.component.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+span.vertical {
+  transform: rotate(270deg);
+  transform-origin: 85% 50%;
+  float: left;
+}
+
+
```
Comment (AlexanderLill) [inline ionic2/app/components/poke-details/poke-attacks.component.scss]: one empty line too many :P

**Review by AlexanderLill — APPROVED** [review]

**Comment by WoH** [comment]
Can we change the background color to the one I used in the Poke(In)Dex?

Edit: Nevermind, this is a different topic, seems fine

File: ionic2/app/components/poke-details/poke-attacks.component.scss (hunk 1 above)
Comment (mfkaptan) [reply ionic2/app/components/poke-details/poke-attacks.component.scss]: fixed in https://github.com/PokemonGoers/Catch-em-all/pull/75/commits/7a627240f8710c2da745da2661c9b658b042c594

**Review by AlexanderLill — APPROVED** [review]

**Pull request merged**